pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[build-system]
```

Comment:
We add protean like this in vfc. 
The same syntax will work with this as well?
``` 
protean = { version = "0.8.0", extras=["sqlite", "postgresql", "elasticsearch", "celery"] }
```

Reply:
Yes, that's correct 👍